src/Form/Mixin/Font.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Kirby\Form\Mixin;
+
+trait Font
+{
+	/**
+	 * Sets the font family (sans or monospace)
+	 */
+	protected string|null $font;
+
+	public function font(): string|null
+	{
+		return $this->font === 'monospace' ? 'monospace' : 'sans-serif';
```

Comment:
`Suggestion:` Maybe we can directly improve its functionality a tidbit.

```suggestion
		return match ($this->font) {
				'monospace', 'mono' => 'monospace',
				default             => 'sans-serif'
		};
```

Reply:
@bastianallgeier what do you think about this?

Reply:
I had already merged this, but it seems like I've wrecked it with a force push after rebase. It's now fixed.